components/Agenda/DownloadAgenda.tsx

```diff
@@ -0,0 +1,205 @@
+  },
+  title: {
+    fontSize: 26,
+    fontWeight: 800,
+    marginBottom: 4,
+  },
+  subtitle: {
+    fontSize: 16,
+    fontWeight: 600,
+    color: '#E74694',
+  },
+  date: {
+    textAlign: 'center',
+    fontSize: 13,
+    marginTop: 6,
+  },
+  tableHeader: {
+    flexDirection: 'row',
+    borderBottomWidth: 1,
+    borderBottomColor: '#E74694',
+    paddingBottom: 6,
+    marginTop: 12,
+  },
+  headerText: {
+    fontSize: 12,
+    fontWeight: 600,
+    color: '#E74694',
+  },
+  row: {
+    flexDirection: 'row',
+    borderBottomWidth: 0.5,
+    borderBottomColor: '#666',
+    paddingVertical: 8,
+  },
+  
+  timeCol: { width: '17%', fontSize: 12, paddingRight: 6 },
+  sessionCol: { width: '50%', paddingHorizontal: 10 },
+  speakerCol: { width: '33%', fontSize: 12, paddingLeft: 6 },
+  sessionTitle: { fontSize: 13, fontWeight: 600, color: '#fff' },
+  sessionType: { fontSize: 10, color: '#bbb' },
+  speakerName: { fontSize: 12, fontWeight: 600, color: '#fff' },
+  speakerTitle: { fontSize: 10, color: '#ccc' },
+ 
+  footer: {
+    position: 'absolute',
+    bottom: 25,
+    left: 0,
+    right: 0,
+    textAlign: 'center',
+    fontSize: 10,
+    color: '#aaa',
+  },
+});
+
+
+const cleanAgenda = (agenda: any[]) => {
+  let tz: string | null = null;
+  const processed = agenda.map((item) => {
+    const tzMatch = item.time.match(/\b(?!AM|PM)([A-Z]{2,4})\b/g);
+    if (tzMatch && !tz) tz = tzMatch[0];
+    return { ...item, time: item.time.replace(/\b(?!AM|PM)([A-Z]{2,4})\b/g, '') };
+  });
+  return { processed, tz };
+};
+
+const AgendaPDF = ({ city }: { city: any }) => {
+  const { processed, tz } = cleanAgenda(city.agenda);
+  return (
+    <Document>
+      <Page style={styles.page}>
+        
+        <View style={styles.header}>
+          <Image src="/img/logos/2025-logo.png" style={styles.logo} />
+          <Text style={styles.title}>
+            {city.name}, {city.country}
+          </Text>
+          <Text style={styles.subtitle}>Conference Agenda</Text>
+          <Text style={styles.date}>
+            {city.date} {tz && `(${tz})`}
+          </Text>
+        </View>
+        
+        <View style={styles.tableHeader}>
+          <Text style={[styles.headerText, { width: '17%' }]}>Time</Text>
+          <Text style={[styles.headerText, { width: '50%' }]}>Session</Text>
+          <Text style={[styles.headerText, { width: '33%' }]}>Speaker</Text>
+        </View>
+     
+        {processed.map((item, i) => (
+          <View style={styles.row} key={i} wrap={false}>
+            <Text style={styles.timeCol}>{item.time}</Text>
+            <View style={styles.sessionCol}>
+              <Text style={styles.sessionTitle}>{item.session}</Text>
+              <Text style={styles.sessionType}>{item.type}</Text>
+            </View>
+            <View style={styles.speakerCol}>
+              {Array.isArray(item.speaker)
+                ? item.speaker.map((id: any, j: React.Key | null | undefined) => {
```

Comment:
`j: React.Key | null | undefined`

This is a redundant line since the speaker id can also be used as a key, they're unique numbers :)